packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.mdx

```diff
@@ -5,9 +5,13 @@ import { RadioButtonGroup } from './RadioButtonGroup';
 
 # RadioButtonGroup
 
-`RadioButtonGroup` is used for selecting single value from multiple options.
+`RadioButtonGroup` is used to select a single value from multiple mutually exclusive options.
 
-Use `RadioButtonGroup` if there are up to four options available. Otherwise use Select component.
+### When to use
+
+Use `RadioButtonGroup` for mutually exclusive selections if there are up to four options available. For a mutually exclusive selection of more than four options, use `Select` component. 
```

Comment:
Why have we decided it's "up to four options"? I am not saying it should be less or more just trying to understand the reasoning.

Reply:
I think the reasoning is a matter of both width of the RadioButtonGroup and avoiding clutter.